misc/tiktok_api.py

```
@@ -1,163 +1,540 @@
+
+                # Fallback: find any file matching the pattern
+                if not actual_path or not os.path.exists(actual_path):
+                    matches = glob.glob(f"{temp_filename}.*")
+                    if matches:
+                        actual_path = matches[0]
+
+                if actual_path and os.path.exists(actual_path):
+                    with open(actual_path, "rb") as f:
+                        return f.read()
+
+            return None
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)